### PR TITLE
When an aspect name is changed all related samples will be removed

### PR DESCRIPTION
**Pull request by harshkothari410**

### CODE REVIEW

File: db/model/aspect.js [hunk 1]
```diff
@@ -203,7 +204,8 @@ module.exports = function aspect(seq, dataTypes) {
       afterUpdate(inst /* , opts */) {
         if (inst.changed('isPublished') &&
           inst.previous('isPublished') &&
-          !inst.getDataValue('isPublished')) {
+          !inst.getDataValue('isPublished') ||
```
Comment (iamigo) [inline db/model/aspect.js]: need db tests for this change

Comment (harshkothari410) [inline db/model/aspect.js]: I am on it.

Comment (harshkothari410) [inline db/model/aspect.js]: @iamigo test added

**Comment by shriramshankar** [comment]
The branch is named "W-3453246" is this fine?

**Pull request merged**